force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -243,7 +275,7 @@ public with sharing class VariablesDataTypesOperators {
         final Double MULTIPLY_FACTOR = 5.0;
         final Double DIVIDE_FACTOR = 9.0;
     
-        return null; // Replace null with the variable you used to store the result
+        return (fahrenheit - 32) * 5 /9;
```

Comment:
Use SUBTRACT_FACTOR, MULTIPLY_FACTOR, and DIVIDE_FACTOR rather than hardcoding the numbers